source/developers/development_environment.markdown

```diff
@@ -22,14 +22,17 @@ Home Assistant is open-source and MIT licensed. The source can be found here:
 You will need to set up a development environment if you want to start developing a new feature or component for Home Assistant. Please follow these steps to get setup.
 Visit the [the Home Assistant repository](https://github.com/home-assistant/home-assistant) first and click fork in the top right.
 
-We suggest that you setup a [virtual environment](https://docs.python.org/3.4/library/venv.html) aka `venv` before running the setup script.
+We suggest that you setup a [virtual environment](https://home-assistant.io/getting-started/installation-virtualenv/) aka [`venv`](https://docs.python.org/3.4/library/venv.html) before running the setup script.
```

Comment:
Ok, not sure I understand 100%, I used those instructions, but will take this out